### DISCUSSION TITLE
When's the next release?

### DISCUSSION THREAD

**Discussion by puyoxyz** [mastodon/mastodon#19761] [General]
There hasn't been a release for, like, over half a year now. So.. when's the next release?

Also, I noticed mastodon.social is running "4.0.0rc1". Where is this version? Don't see it on the Releases page or on the tags.

![image](https://user-images.githubusercontent.com/15311104/200106744-214ac4a3-bb6f-44d5-8af3-b6c9642b072b.png)

**Comment by trwnh** [comment]
The release should be tagged once some showstopping bugs are fixed. Think of the version on mastodon.social as a "pre-release" -- it is running the code in the `main` branch, before it gets tagged as a release.

**Comment by chris-ca** [comment]
4.0.0rc1 and 4.0.0rc2 have just been tagged on the releases page: https://github.com/mastodon/mastodon/releases
